### DISCUSSION TITLE
Retry the request according to the customized status code

### DISCUSSION THREAD

**Discussion by xlg-go** [apache/apisix#5621] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/29967092/143543144-c77f5e5f-a331-4c40-85d7-acf0a51c9c31.png)

Can I retry the request based on the customized http status code？

**Comment by tokers** [comment, marked as answer]
Currently, the status code from the last upstream endpoint is not considered to decide the retry behavior.

**Reply by xlg-go** [reply]
> Currently, the status code from the last upstream endpoint is not considered to decide the retry behavior.

ok~thanks a lot
